File: B/php/FootballTeam.php
<?php
class FootballTeam
{
    private $name;
    private $origin;
    private $year;
    private $players = array();

    // set default data
    public function __construct($name = "-", $origin = "-", $year = "-", $players = array())
    {
        $this->name = $name;
        $this->origin = $origin;
        $this->year = $year;
        $this->players = $players;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function getName()
    {
        return $this->name;
    }

    public function setOrigin($origin)
    {
        $this->origin = $origin;
    }

    public function getOrigin()
    {
        return $this->origin;
    }

    public function setYear($year)
    {
        $this->year = $year;
    }

    public function getYear()
    {
        return $this->year;
    }

    public function setPlayers($players)
    {
        $this->players = $players;
    }

    public function getPlayers()
    {
        return $this->players;
    }

    public function addPlayer($player)
    {
        $this->players[] = $player;
    }

    public function showTeam()
    {
        echo "Team Name      : " . $this->getName() . "<br>";
        echo "Origin Country : " . $this->getOrigin() . "<br>";
        echo "Year of Build  : " . $this->getYear() . "<br>";

        $players = $this->getPlayers();
        for($i = 0; $i < count($players); $i++)
        {
            echo "- " . $players[$i] . "<br>";
        }
    }
}
?>

File: B/python/FootballTeam.py
class FootballTeam:
    name = ""
    origin = ""
    year = ""
    players = []

    # set default data
    def __init__(self, name = "-", origin = "", year = "", players = []):
        self.name = name
        self.origin = origin
        self.year = year
        self.players = players

    def setName(self, name):
        self.name = name

    def getName(self):
        return self.name

    def setOrigin(self, origin):
        self.origin = origin

    def getOrigin(self):
        return self.origin

    def setYear(self, year):
        self.year = year

    def getYear(self):
        return self.year

    def setPlayers(self, players):
        self.players = players

    def getPlayers(self):
        return self.players

    def addPlayer(self, player):
        self.players.append(player)

    def showTeam(self):
        print("Team Name      : " + self.getName())
        print("Origin COuntry : " + self.getOrigin())
        print("Year of Build  : " + self.getYear())

        players = self.getPlayers()
        for i in range(len(players)):
            print("    - " + players[i])